### PR TITLE
minor fix base on RP's feedback

### PR DESCRIPTION
**Pull request by taoshengshi**
Please kindly review the fix based on feedback from RP.

### CODE REVIEW

File: lilypad/lilypad-milky-way-reference/run-a-node.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ systemd is a system and service manager for Linux operating systems. systemd ope
 To install systemd, open `/etc/systemd/system/bacalhau.service` in your preferred editor:
 
 {% hint style="info" %}
-Hint: `sudo editor /etc/systemd/system/bacalhau.service`
+Hint: `sudo vim /etc/systemd/system/bacalhau.service`
```
Comment (AquiGorka) [inline lilypad/lilypad-milky-way-reference/run-a-node.md:129]: Interesting, I thought the `.idea` file that you added is meant for `webstorm` but here you go with `vim`?
In any case, I think we should keep `editor` as that is a generic solution, as opposed to trying to force using `vim`.

File: lilypad/lilypad-milky-way-reference/run-a-node.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ WantedBy=multi-user.target
 Open `/etc/systemd/system/lilypad-resource-provider.service` in your preferred editor.
 
 {% hint style="info" %}
-Hint: `sudo editor /etc/systemd/system/lilypad-resource-provider.service`
+Hint: `sudo Vim /etc/systemd/system/lilypad-resource-provider.service`
```
Comment (AquiGorka) [inline lilypad/lilypad-milky-way-reference/run-a-node.md:155]: nit: lowercase `vim`

**Review by AquiGorka — APPROVED** [review]
Thank you @taoshengshi , can you set the correct lowercase for the second hint before merging?

File: lilypad/lilypad-milky-way-reference/run-a-node.md (hunk 2 above)
Comment (taoshengshi) [reply lilypad/lilypad-milky-way-reference/run-a-node.md]: fixed.

**Review by richardbremner — APPROVED** [review]
lgtm

**Pull request merged**